src/argilla/client/feedback/schemas/remote/records.py

```diff
@@ -117,37 +120,29 @@ class RemoteFeedbackRecord(FeedbackRecord, RemoteSchema):
             question. Defaults to an empty list.
     """
 
+    # TODO: remote record should receive a dataset instead of this
     question_name_to_id: Optional[Dict[str, UUID]] = Field(..., exclude=True, repr=False)
 
     responses: List[RemoteResponseSchema] = Field(default_factory=list)
-    suggestions: Union[Tuple[RemoteSuggestionSchema], List[RemoteSuggestionSchema]] = Field(
-        default_factory=tuple, allow_mutation=False
-    )
+    suggestions: Union[Tuple[AllowedSuggestionSchema], List[AllowedSuggestionSchema]] = Field(default_factory=tuple)
 
     class Config:
+        allow_mutation = True
         validate_assignment = True
 
-    def __update_suggestions(
+    def __normalize_suggestions_to_update(
```

Comment:
I think we should carefully review the `suggestions` update/addition workflow, because I think we're adding too much complexity here that can probably be simplified

Reply:
yes, but separate PR. I didn't change the internal logic, just separated the method in 2 different ones.